src/main/java/com/greenfoxacademy/goddesstribesbackend/controllers/BattleController.java

```diff
@@ -13,8 +13,8 @@
 @RestController
```

Comment:
I think these files are updated in the latest `master`, also it is not a good practice to add and commit files not related to the Jira ticket (in this situation for example not the leaderboard files) -so basically we need just the files related to Leaderboard

Reply:
Hi, this branch has been launched from the TGB14 since that was the next in line with a pull request. Only 5 files were added those related to the TGB7.  It seems that Git added all the files from the TGB14 too.  While I completely agree with your comment, I do not know what did I wrong.